CMakeLists.txt

```diff
@@ -39,14 +38,23 @@ function(packageProject)
     add_library(${PROJECT_NAMESPACE}${PROJECT_NAME} ALIAS ${PROJECT_NAME})
   endif()
 
-  if(DEFINED PROJECT_VERSION_HEADER)
+  if(DEFINED PROJECT_VERSION_HEADER OR DEFINED PROJECT_EXPORT_HEADER)
```

Comment:
I don't think this should be combined here. Just move `PROJECT_VERSION_INCLUDE_DIR` out and define the conditionals separately.

Reply:
That is simply not true!